### PR TITLE
responcive page

### PR DESCRIPTION
**Pull request by ali1373107**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2,20 +2,96 @@
...
 </head>
-
 <body>
-
-    <!-- Add your markup here -->
-
-
-    <!-- Please leave this in to credit the creators of this design -->
-    <p>This website design was created by Wix.com, and is used here for strictly educational purposes.</p>
+    <header>
```
Comment (Hamisakim) [inline index.html:11]: keep nav bar seperate from header

File: index.html [hunk 2]
```diff
@@ -2,20 +2,96 @@
...
+                 <div class="buttons">
+                    <button class="btn11 btnn">RESUME</button>
+                    <button class="btn-22 btnn">PROJECTS</button>
+                </div>
+                 <p>I’m a g213reat place for you to tell a story and let your users 
+                    know a little more about you.</p>
+            </section>
+            
+        </main>
+        <footer>
```
Comment (Hamisakim) [inline index.html:73]: keep footer outside content

**Comment by ali1373107** [comment]
Hi Sami this is the latest commit that I have made

**Comment by ali1373107** [comment]
I havent added hover to elements yet

**Comment by Hamisakim** [comment]
good work @ali1373107.  Going forwards lets make sure to focus more little bits of styline. Like the font, this will make the website much nicer to look at.

**Pull request closed without merging**